disk/disk.go

```diff
@@ -154,6 +154,15 @@ func (d *Disk) Remove(name string, all bool) error {
 	return os.RemoveAll(name)
 }
 
+func (d *Disk) Sync(name string) error {
```

Comment:
@tnachen I am not sure if we need that strong guarantee. I can image that I would like to write to three files into the buffer, sync them together. If that sync succeeds, I know all my previous writes have been persistent. If the sync fails (or it succeeds but the reply get lost), I can simply redo. I do not have to know exactly if it succeeds or not. And I do not need to abort anything in between.